module/Api/src/Domain/CommandHandler/Application/ReviveApplication.php

```diff
@@ -39,6 +43,8 @@ public function handleCommand(CommandInterface $command)
 
         $result = new Result();
 
+        $licence = $application->getLicence()->getId();
```

Comment:
M: Looks like `->getId()` shouldn't be here as the code below all uses `$licence->getId()`